nova/virt/vmwareapi/vmops.py

```diff
@@ -2156,84 +2340,75 @@ def finish_revert_migration(self, context, instance, network_info,
             finally:
                 self._attach_volumes(instance, block_device_info, adapter_type)
 
-        if power_on:
```

Comment:
Why shouldn't it be powered on, when it is requested so?

Reply:
This was part of `finish_revert_migration()` and still exists [there](https://github.com/sapcc/nova/pull/173/files#diff-1e2677712ffcba6684bf10f3ceada016e5bea75e864b3572df95358207f67c26R2276) (check above)